pkg/instancegroups/instancegroups.go

```diff
@@ -137,7 +143,16 @@ func (r *RollingUpdateInstanceGroup) RollingUpdate(rollingUpdateData *RollingUpd
 		}
 	}
 
-	for _, u := range update {
+	settings := resolveSettings(cluster, r.CloudGroup.InstanceGroup)
+
+	for uIdx, u := range update {
+		if featureflag.ConfigurableRollingUpdate.Enabled() && *settings.TaintAllNeedUpdate {
```

Comment:
Currently kops doesn't uncordon a node on a failed drain. The only automated way to recover is to do another rolling update to complete the update.

In case the instance spec is reverted, we might add code to ensure that nodes with our taint are considered NotReady and thus updated on a subsequent rolling update. But this feature is likely to be only enabled on instancegroups with cluster autoscaler enabled, so that will eventually remove them for being underused.